src/api/controllers/projectController.py

```diff
@@ -1,60 +1,80 @@
-from api.models import db, Project, UserHasProject, ProjectLink, ProjectFeedback
+from api.models import db, User, Project, UserHasProject, ProjectLink, ProjectFeedback
 
 class ProjectController:
-    session = db.session()
+    # TODO: remove this (also from userController)?
+    session = db.session
 
     # Project
-    def create_project(self, **kwargs):
+    def create_project(self, user_id, **kwargs):
         try:
+            user = User.query.filter_by(id=user_id).first()
+
+            if user == None:
+                return None, "User not found", 404
+
             project = Project(**kwargs)
-            self.session.add(project)
+
+            userHasProject = UserHasProject(role=1)
+            userHasProject.project = project
+            userHasProject.user = user
```

Comment:
`userHasProject` table is not filled automatically?

Reply:
I followed the example in the documentation: https://docs.sqlalchemy.org/en/13/orm/basic_relationships.html#association-object

Didn't find any other way to create the relationship.